R/proj.R

```diff
@@ -17,6 +17,31 @@ proj_find <- function(path = ".") {
 
 is_proj <- function(path = ".") !is.null(proj_find(path))
 
+is_package <- function(base_path = proj_get()) {
+  res <- tryCatch(
+    rprojroot::find_package_root_file(path = base_path),
+    error = function(e) NA
```

Comment:
I think it's slightly better to use `NULL` in situations like this, because `NA` is a vector.